x-pack/metricbeat/module/aws/ec2/_meta/data.json

```diff
@@ -47,31 +51,37 @@
             },
             "network": {
                 "in": {
-                    "bytes": 85.2,
-                    "packets": 1.4
+                    "bytes": 1297480,
+                    "bytes_per_sec": 4324.933333333333,
+                    "packets": 2734.8
```

Comment:
Should we also calculate `packets_per_second`?

Reply:
Yes, ui team also brought this up. I will add it in next PR.